app/views/public_pages/faq.html.erb

```diff
@@ -19,7 +19,12 @@
             <h3><%=t("views.public_pages.faq.getting_started.non_filer.q") %></h3>
           </a>
           <div class="accordion__content" id="a1">
-            <p><%=t("views.public_pages.faq.getting_started.non_filer.a") %></p>
+            <p>
+              <%= t(
+                "views.public_pages.faq.getting_started.non_filer.a",
+                years: [TaxReturn.current_tax_year - 3, TaxReturn.current_tax_year - 2, TaxReturn.current_tax_year - 1].to_sentence(locale: locale)
```

Comment:
Can also use `TaxReturn.filing_years.without(TaxReturn.current_tax_year)`. If you need them with oldest year first, append `.reverse`

Reply:
Actually, there's a method on TaxReturn that does this already:

`TaxReturn.backtax_years.reverse.to_sentence(locale: locale)`